src/lib/queries/lexical-query-lookup.js

```diff
@@ -0,0 +1,73 @@
+import LexicalQuery from './lexical-query.js'
+import Query from './query.js'
+import { Constants } from 'alpheios-data-models'
+
+import {LemmaTranslations} from 'alpheios-lemma-client'
+import {AlpheiosTuftsAdapter} from 'alpheios-morph-client'
+import {Lexicons} from 'alpheios-lexicon-client'
+
+import ResourceOptions from '../options/resource-options'
+
+/**
+   * It is a subclass of the LexicalQuery class created for lookup component (lookup.vue).
+   * It is created for initiating adapters inside it (similiar to embedded)
+   */
+
+export default class LexicalQueryLookup extends LexicalQuery {
+  /**
+   * @createForLookup - it is used for rendering popup/panel with data for the current text
+   * @param {TextSelector} textSelector - text selector containg the word for getting data from adapters and showing in the popup/panel
+   * @param {UIController} uiController - ui controller for re-rendering popup/panel (passed fro current rendered popup)
+   */
+
+  static createForLookup (textSelector, uiController) {
```

Comment:
Sure, fixed

Reply:
Can we rename `static createForLookup (textSelector, uiController)` to `static create (textSelector, uiController)` so that it will match the `create` method defined in other Query objects and work within an inheritance tree?